src/utils/parser.js

```diff
@@ -75,21 +75,30 @@ function getFunctionName(path) {
     return parent.key.name;
   }
 
+  if (parent.type == "ObjectExpression" || path.node.type == "ClassMethod") {
+    return path.node.key.name;
+  }
+
   if (parent.type == "VariableDeclarator") {
     return parent.id.name;
   }
 
   return "anonymous";
 }
 
+function isFunction(path) {
+  return t.isFunction(path) || t.isArrowFunctionExpression(path) ||
```

Comment:
Could likely make it more exhaustive but I found that it worked for the different types of function declarations I'm aware of.